source/edge-server/configure.txt

```diff
@@ -10,4 +10,307 @@ Configure Edge Server
    :depth: 2
    :class: singlecol
 
-This is a placeholder page.
+Edge Server is a "local" server that sits between your client devices and 
+the MongoDB Atlas App. For most use cases, you'll provision hardware 
+on-premises to host the Edge Server. While you're developing your Edge 
+Server application, you may host it on a cloud server for convenience.
+
+Client devices on-prem connect to Edge Server. Your Edge Server
+connects to a MongoDB Atlas App. Clients do not connect to the MongoDB 
+Atlas App directly.
```

Comment:
[suggestion]
add that simplified diagram here

Reply:
If we want to do this, we should get the image asset directly. I only have a screenshot I took from a Google doc. I'll see if I can track that down.

Reply:
Added it!